docs/requirements.txt

```
@@ -1,5 +1,6 @@
 cirq-superstaq[dev]~=0.5.6
 ipywidgets>=8.0.0
 nbsphinx>=0.9.0
+pandoc~=2.3.0
```

version should just be [2.3](https://pypi.org/project/pandoc/) i think (and also we should use `>=` if possible)
```suggestion
pandoc>=2.3
```